src/main/java/com/opinta/util/LogMessageUtil.java

```diff
@@ -44,6 +46,10 @@ public static String getAllByFieldOnErrorLogEndpoint(Class clazz, Class clazzFie
     public static String getByIdOnErrorLogEndpoint(Class clazz, Object value) {
         return format("%s %s not found!", clazz.getSimpleName(), value);
     }
+    
+    public static String getByUserOnErrorLogEndpoint(Class clazz, User user) {
```

Comment:
Try to do this way.
Do not create new method.
Instead use existent.
1. Rename existent method getAllByFieldOnErrorLogEndpoint -> getByFieldOnErrorLogEndpoint
2. Use it like this
getByFieldOnErrorLogEndpoint(Counterparty.class, User.class, userValue);